src/util.rs

```diff
@@ -0,0 +1,39 @@
+use crate::execution_class::ExecutionClass;
+use regex::Regex;
+
+/// Extract exception info from stderr
+///
+/// # Arguments
+///
+/// * `stderr_list` - lines of stderr
+///
+/// # Return value
+///
+/// Exception info as a `ExecutionClass` struct
+pub fn exception_from_stderr(stderr_list: &[String]) -> Option<ExecutionClass> {
```

Comment:
maybe we should name file cpp.rs and rename function to specify that it is a cpp exception

Reply:
exception_from_stderr -> cpp_exception_from_stderr

Reply:
util.rs will be used for rust panic too